Mathlib/Analysis/Calculus/FDeriv/ContinuousAlternatingMap.lean

```diff
@@ -0,0 +1,246 @@
+/-
+Copyright (c) 2025 Yury Kudryashov. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yury Kudryashov
+-/
+import Mathlib.Analysis.Calculus.FDeriv.ContinuousMultilinearMap
+import Mathlib.Analysis.NormedSpace.Alternating.Basic
+
+/-!
+# Derivatives of operations on continuous alternating maps
+
+In this file we prove formulas for the derivatives of
+
+- `ContinuousAlternatingMap.compContinuousLinearMap`, the pullback of a continuous alternating map
+  along a continuous linear map;
+- application of a `ContinuousAlternatingMap` as a function of both the map and the vectors.
+-/
+
+variable {𝕜 ι E F G H : Type*}
+  [NontriviallyNormedField 𝕜]
+  [NormedAddCommGroup E] [NormedSpace 𝕜 E] [NormedAddCommGroup F] [NormedSpace 𝕜 F]
+  [NormedAddCommGroup G] [NormedSpace 𝕜 G] [NormedAddCommGroup H] [NormedSpace 𝕜 H]
+
+open ContinuousAlternatingMap
+open scoped Topology BigOperators
+
+section CompContinuousLinearMap
+
+variable
+  {f : E → G [⋀^ι]→L[𝕜] H} {f' : E →L[𝕜] G [⋀^ι]→L[𝕜] H}
+  {g : E → F →L[𝕜] G} {g' : E →L[𝕜] F →L[𝕜] G}
+  {s : Set E} {x : E}
+
+/-!
+### Derivative of the pullback
+
+In this section we prove a formula for the derivative
+of the pullback of a continuous alternating map along a continuous linear map,
+as a function of both maps.
+-/
+
+theorem ContinuousAlternatingMap.hasStrictFDerivAt_toContinuousMultilinearMap_comp_iff [Finite ι] :
+    HasStrictFDerivAt (toContinuousMultilinearMap ∘ f) (toContinuousMultilinearMapCLM 𝕜 ∘L f') x ↔
+      HasStrictFDerivAt f f' x := by
+  cases nonempty_fintype ι
+  constructor <;> intro h
+  · rw [hasStrictFDerivAt_iff_isLittleOTVS] at h ⊢
+    refine Asymptotics.IsBigOTVS.trans_isLittleOTVS ?_ h
+    simp only [Function.comp_apply, ← toContinuousMultilinearMapCLM_apply 𝕜,
+      ContinuousLinearMap.comp_apply, ← map_sub]
+    apply LinearMap.isBigOTVS_rev_comp
+    simp [isEmbedding_toContinuousMultilinearMap.nhds_eq_comap]
+  · exact (toContinuousMultilinearMapCLM 𝕜).hasStrictFDerivAt.comp x h
+
+section HasFDerivAt
+
+variable [Fintype ι] [DecidableEq ι]
+
+theorem ContinuousAlternatingMap.hasStrictFDerivAt_compContinuousLinearMap
+    (fg : (G [⋀^ι]→L[𝕜] H) × (F →L[𝕜] G)) :
+    HasStrictFDerivAt
+      (fun fg : (G [⋀^ι]→L[𝕜] H) × (F →L[𝕜] G) ↦ fg.1.compContinuousLinearMap fg.2)
+      (compContinuousLinearMapCLM fg.2 ∘L .fst _ _ _ +
+        fg.1.fderivCompContinuousLinearMap fg.2 ∘L .snd _ _ _)
```

Comment:
untested
```suggestion
      (compContinuousLinearMapCLM fg.2 ∘L .fst .. +
        fg.1.fderivCompContinuousLinearMap fg.2 ∘L .snd ..)
```

Reply:
Even if it works, it isn't clear right away whether `..` should add one extra `_` for the application of `.fst _ _ _`.